frontend/public/components/_scrollbars.scss

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
My preference is to not style the scrollbar.  @spadgett?

Reply:
We might pull this change out into a separate PR for discussion.

@sg00dwin found a way for the styled scrollbar to only show up on hover, which addresses my primary concern with it. Although I'm not sure what the behavior is on mobile or if you have scrollbars always visible.

Reply:
@spadgett  Behavior on mobile... ios defaults to using the iphone's scroll handle which is only shown on touch. On android the custom scroll handle is shown when `.navigation-container` detects touch.